File: DSA/Java Practice/KthSmallest.java
import java.util.*;
import java.util.Arrays;

public class KthSmallest {
    public static void main(String[] args) {
        int k,n,i;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the value of k: ");
        k=sc.nextInt();
        System.out.println("Enter the length of array: ");
        n=sc.nextInt();
        int[] arr = new int[n];
        
        

        for(i=0;i<n;i++)
        {
            System.out.println("Enter the value at position "+i+" : ");
            arr[i]=sc.nextInt();
        }

        

        Arrays.sort(arr);

        for(i=0;i<n;i++)
        {
            System.out.println("Value at position "+i+" is "+arr[i]);
        }

        System.out.println("Kth smallest element is: "+arr[k-1]);


    }
}

File: DSA/Python practice/kthsmallest.py
arr=[]

arrLen = int(input("\nEnter the length of array: "))
k=int(input("\nenter the value of k: "))

for i in range(0,arrLen):
    num = int(input("\nEnter the value at position "+str(i)+" : "))
    arr.append(num)
arr.sort()
print("Sorted array is: ",arr)
print("Kth smallest number is: ",arr[k-1])


